src/components/Game.jsx

```diff
@@ -32,7 +32,11 @@ class Game extends Component {
   }
 
   calculateScore(word) {
-    return word.length
+    if (word.length >= 3 && word.length <= 8) {
```

Comment:
Yes but for a more robust function, I think a safeguard on this function would be better.  If unit tests were made for this function, it would also need to handle edge cases.  There might be places in the code where the word length is checked but code changes all the time.